sdk/core/azure-core/src/main/java/com/azure/core/util/polling/implementation/PollingConstants.java

```diff
@@ -63,4 +63,8 @@ public final class PollingConstants {
      * The polling status indicating the long running operation has succeeded.
      */
     public static final String STATUS_SUCCEEDED = "Succeeded";
+    /**
+     * The polling status indicating the long running operation has been cancelled.
+     */
+    public static final String STATUS_CANCELLED = "Canceled";
```

Comment:
`Canceled` is also a complete status
https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-operations

Reply:
So, canceled worked accidentally because we were mapping unknown status to complete?

Reply:
Yes, probably. Though "canceled" is not widely used in LRO, and we didn't had many codegen LRO in the first place.

And currently, the "agrifood" is the only one we can see that treated the status wrong (take "Waiting" as terminal).